src/styles/global/variables/_color.scss

```diff
@@ -0,0 +1,12 @@
+// Color palette -- All color variables should start with color-
+// ---------------------------------------------------
+
+$color: (
+  '--color-black': #000,
+  '--color-blueprint': #0065f2,
```

Comment:
For the `--color-blueprint-*` variables is the intention to find/replace them when starting a project? 

I ask because I feel `blueprint` makes sense now but once you get into building possibly `theme` or something else.